backend/src/images/messageImages.ts

```diff
@@ -16,8 +16,6 @@ const errorTheiaDark = "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbm
 
-const errorTheiaLight = "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4NCjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+DQo8c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSIxNyIgaGVpZ2h0PSIxNyIgdmlld0JveD0iMCAwIDE3IDE3Ij4NCiAgPGRlZnM+DQogICAgPGNsaXBQYXRoIGlkPSJjbGlwLWVycm9yX3RoZWlhX2xpZ2h0Ij4NCiAgICAgIDxyZWN0IHdpZHRoPSIxNyIgaGVpZ2h0PSIxNyIvPg0KICAgIDwvY2xpcFBhdGg+DQogIDwvZGVmcz4NCiAgPGcgaWQ9ImVycm9yX3RoZWlhX2xpZ2h0IiBjbGlwLXBhdGg9InVybCgjY2xpcC1lcnJvcl90aGVpYV9saWdodCkiPg0KICAgIDxnIGlkPSJHcm91cF8zNTMxIiBkYXRhLW5hbWU9Ikdyb3VwIDM1MzEiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0xIC0xLjAwNCkiPg0KICAgICAgPHBhdGggaWQ9IlBhdGhfMTYwOCIgZGF0YS1uYW1lPSJQYXRoIDE2MDgiIGQ9Ik0yMzAuMzUxLTEwMi40YTguNSw4LjUsMCwwLDAtOC41LDguNSw4LjUsOC41LDAsMCwwLDguNSw4LjUsOC41LDguNSwwLDAsMCw4LjUtOC41LDguNSw4LjUsMCwwLDAtOC41LTguNVptMy41NTQsMTIuMzc2YS43OTMuNzkzLDAsMCwxLS41NTkuMjI3LjguOCwwLDAsMS0uNTY3LS4yMzdsLTIuNDQtMi40NzMtMi41NTgsMi42MjdhLjc5My43OTMsMCwwLDEtLjU2Ny4yNDEuNzkzLjc5MywwLDAsMS0uNTU2LS4yMjcuNzkzLjc5MywwLDAsMS0uMjQzLS41NjEuNzkzLjc5MywwLDAsMSwuMjI4LS41NjdsMi41ODItMi42NS0yLjM2NS0yLjRhLjc5My43OTMsMCwwLDEtLjIzMS0uNTY1Ljc5My43OTMsMCwwLDEsLjIzOS0uNTYyLjc5My43OTMsMCwwLDEsLjU2NS0uMjMxLjc5My43OTMsMCwwLDEsLjU2Mi4yMzlsMi4zNDMsMi4zNzMsMi4zMDktMi4zNzFhLjc5NC43OTQsMCwwLDEsLjU2LS4yNDIuNzkzLjc5MywwLDAsMSwuNTY2LjIyNy43OTMuNzkzLDAsMCwxLC4yNDMuNTYxLjc5My43OTMsMCwwLDEtLjIyOC41NjdsLTIuMzMxLDIuMzkyLDIuNDYyLDIuNDkzYS43OTMuNzkzLDAsMCwxLC4yMjkuNTY3Ljc5My43OTMsMCwwLDEtLjI0My41NjJaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMjIwLjg1MSAxMDMuNDA0KSIgZmlsbD0iI2U4NGQ0ZCIvPg0KICAgIDwvZz4NCiAgPC9nPg0KPC9zdmc+DQo="
-
 export function getImage(state: number, isInBAS: boolean) {
```

Comment:
Its better remark it. So we dont have to search for it in case we  will need it.

Reply:
Done